Plutarch/Builtin.hs

```diff
@@ -3,6 +3,7 @@
 {-# OPTIONS_GHC -Wno-orphans #-}
 
 module Plutarch.Builtin (
+  Flip,
```

Comment:
.…This should not be public API.

Reply:
maybe we should add it to `Extra.TryFrom`, otherwise everybody declares their own Flip? Maybe it's not an issue though.

Reply:
it really does feel weird for everyone to have to define it inline - even if it's supposed to be a "type lambda".

+1 to having it exposed in `plutarch-extra` at this point